RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
+  the top of the darkroom view to replace the information previously overlaid on
+  the histogram. The specific information shown can be configured in preferences.
+
+- The 'tagging' module is faster, and can now display hierarchical tags in a
+  tree view. Tags can now be designated as 'private' (not exported by default),
+  'synonym' (to help search engines), and 'category' (not exported, for organizing
+  the tag library). The metadata exported with images can now be configured in the
+  'export selected' module, allowing you to choose which main types are exported,
+  as well as define values for specific tags based on formulas.
+  
 - Many code optimizations for CPU and SSE paths.
 
 - A new preference to expand/collapse a darkroom module when it is
   activated/deactivated.
 
-- The collect module supports single click if the corresponding new
-  option is activated.
-
-- The collect module supports range selection (for numeric/dates)
-  when using single click mode.
-
-- The gradient mask orientation is now clearly displayed by using an
-  arrow.
+- The 'collect images' module has a new single-click option featuring range selection
+  for date-time and numeric values.
+  
+- The orientation of drawn masks using the 'gradient' shape is now clearly displayed
+  with an arrow.
 
-- Add support for theme, and allow to dynamically load theme from the
-  preferences.
+- The 'graduated density' module has better accuracy when computing the
+  rotation to avoid sporadic flipping of the gradient.
 
-- The graduatednd module has better accuracy when computing the
-  rotation to avoid the gradient to flip sporadically.
-
-- Make sure the OpenCL kernel code are recompiled when the driver
+- Make sure the OpenCL kernel code is recompiled when the driver
   version is updated.
 
-- Add color pickers in the split toning module.
-
-- Add color pickers in the graduatednd module.
-
-- Add color picker to the watermark module.
+- Add color pickers for 'split toning', 'graduated density' and 'watermark' modules.
 
-- The color picker point positions are kept during the editing on the
-  module.
+- The color picker positions are kept during editing within a module.
 
-- The map view will zoom on the images of the selected collection.
+- The map view can zoom on the images of the selected collection.
 
-- The slideshow supports changing the delay between each images.
+- The slideshow will now start at the selected images if any, and supports
+  changing the delay between images. It will also now be more responsive when
+  manually moving backward and forward through the images.
 
-- The slideshow will now start at the selected pictures if any.
-
-- The slideshow will now be more responsive when changing the picture
-  backward and forward.
-
-- A new module 'basic adjustments' has been added. It allows to adjust
+- A new 'basic adjustments' module has been added. It allows to adjust
   the exposure, highlight compression, contrast, brightness and saturation.
```

Comment:
There are 7 sliders in the module, 5 have been mentioned.

Reply:
Awww, you mean I have to actually look at that module?!  ;-)

Reply:
I went through the referenced modules in the release notes to see if the changes here are in modules ;)